src/App/GetStarted/get-started/ForDevelopers/index.js

```diff
@@ -103,6 +134,10 @@ const Installation = () => (
                     </div>
                 ))}
             </div>
+            <div className="alert alert-informative mt-4">
+                <i className="material-icons alert-icon" aria-hidden="true">info</i>
+                <p><b>Target Browser:</b> "Last 3 versions, > 1%, not dead"</p>
```

Comment:
Don't understand " > 1%, not dead" thing 😓

Reply:
this is our browserlist setting. It is used by polyfills, transpilers (e..g. Babel) & autoprefixer (for CSS) to ensure the code can be used by the browsers we are targeting

"> 1%" -> browser's releases used by more than 1% of users (based on general statistics, not our own stats)
"not dead" -> browsers not dead. i.e. not IE whatever release, not the old Edge pre-chromium, ...

Reply:
Aha, great 👍